### PR TITLE
fix: handle errors from Tesseract

### PR DESCRIPTION
**Pull request by awalker4**
Certain regions of a document are failing ocr with this error:

`pytesseract.pytesseract.TesseractError: (-8, 'Estimating resolution as 1250')` (for some value)

When I try the same region on the CLI, I get:

```
$ tesseract bad_tile.jpeg output
Estimating resolution as 1813
Floating point exception
```

Whatever the root cause, let's catch this error and return an empty string.

### CODE REVIEW

**Comment by awalker4** [comment]
Looks like those ingest tests have been failing on main for a while, so I'm assuming we can ignore here.

**Comment by cragwolfe** [comment]
> Looks like those ingest tests have been failing on main for a while, so I'm assuming we can ignore here.

Yeah, that just indicates the current version of unstructured-inference would produce different outputs if the ingests tests are run. Hopefully, they are good different outputs!

**Review by shreyanid — APPROVED** [review]
Code looks good to me! Tested in docker to reproduce the issue, and this change results in a 200 status for otherwise failing documents.

File: unstructured_inference/inference/elements.py [hunk 1]
```diff
@@ -269,7 +269,11 @@ def ocr(text_block: TextRegion, image: Image.Image, languages: str = "eng") -> s
     agent = tesseract.ocr_agents.get(languages)
     if agent is None:
         raise RuntimeError("OCR agent is not loaded for {languages}.")
-    return agent.detect(cropped_image)
+
+    try:
+        return agent.detect(cropped_image)
+    except tesseract.TesseractError:
```
Comment (cragwolfe) [inline unstructured_inference/inference/elements.py:275]: add a `logger.warn("<something", exc_info=True)`

**Pull request merged**